build/buildBuildImages.sh

```diff
@@ -397,6 +397,45 @@ function buildCliImage() {
 	echo "$builtImageName" >> $ACR_BUILD_IMAGES_ARTIFACTS_FILE
 }
 
+function buildAndRunImage() {
+	buildBuildScriptGeneratorImage
+	
+	local debianFlavor=$1
+	local devImageTag=BuildAndRun
+	local builtImageName="$ACR_BUILD_AND_RUN_IMAGE_REPO"
+
+	if [ -z "$debianFlavor" ] || [ "$debianFlavor" == "stretch" ]; then
+		debianFlavor="stretch"
+	elif  [ "$debianFlavor" == "buster" ]; then
```

Comment:
From the PR, I assume that Bullseye wasn't an easy change?

Reply:
Correct, we'll have to upgrade to golang 1.17+

Reply:
Correct, we'll have to upgrade to golang 1.17+ which will also require us to migrate away from `go dep`  to using `go.mod`